lib/raven/processor/removecircularreferences.rb

```diff
@@ -3,10 +3,12 @@ class Processor::RemoveCircularReferences < Processor
     def process(v, visited = [])
       return "(...)" if visited.include?(v.__id__)
       visited += [v.__id__]
-      if v.is_a?(Hash)
-        v.each_with_object({}) { |(k, v_), memo| memo[k] = process(v_, visited) }
+      if v.frozen?
```

Comment:
Modifying this so that it continues to process frozen collections (with possible not-frozen submembers) doesn't significantly impact perf, so I'll do that.